site/src/sphinx/server-grpc.rst

```diff
@@ -243,6 +243,26 @@ to all your clients before enabling this flag!
 
 See more details at :ref:`client-grpc`.
 
+Server Reflection
+=================
```

Comment:
Needs to use `----------------` because this is a subsection.

Reply:
Oops - guess Exception Propagation was wrong too

Reply:
Yeah, but I fixed it in other PR. :-)